packages/cosmos/src/types.ts

```diff
@@ -26,6 +26,22 @@ export interface Database extends PersistedResource {
   _users: 'users/';
 }
 
+export interface PartitionKeyRange extends PersistedResource {
+  id: string;
+  minInclusive: string;
+  maxExclusive: string;
+  ridPrefix: number;
+  throughputFraction: number;
+  status: string;
+  parents: unknown[];
+}
+
+export type PartitionKeyRanges = {
```

Comment:
I think the PartitionKeyRanges object will be of type PersistedResource
https://docs.microsoft.com/en-us/rest/api/cosmos-db/get-partition-key-ranges#body-1
```ts
export interface PartitionKeyRanges extends PersistedResource {
  PartitionKeyRanges: PartitionKeyRange[];
   _count: number;
}
```